cost/aws/ec2_compute_optimizer/README.md

```diff
@@ -0,0 +1,94 @@
+# AWS EC2 Compute Optimizer Recommendations
+
+## What It Does
+
+This policy template reports EC2 rightsizing recommendations produced by the [AWS Compute Optimizer](https://aws.amazon.com/compute-optimizer/faqs/#EC2_instance_recommendations). Optionally, this report can be emailed and the offending instances can be resized.
```

Comment:
Maybe we should add some detail explaining that there may be overlap between the recommendations produced by this policy and AWS Rightsize EC2 Instances policy. 

Could be worth catching up with Jeremy as to whether this should be in the Catalog as may cause confusion from customers as to which policy to run